.gitignore

```diff
@@ -254,3 +254,6 @@ dist
 # End of https://www.toptal.com/developers/gitignore/api/node,maven,java,intellij+all
 
 .DS_Store
+
+
+DBConfig.java
```

Comment:
Remove this and add the DBconfig to the repo

**MAKE SURE TO REMOVE THE USERNAME AND PASSWORD**